examples/Jenkinsfile

```diff
@@ -0,0 +1,56 @@
+pipeline {
+    
+    agent any
+    
+    environment {
+        // Replace the <tags> below with your values.
+        FALCON_CLIENT_SECRET = credentials('FALCON_CLIENT_SECRET')
+        FALCON_CLIENT_ID = credentials('FALCON_CLIENT_ID')
+        GITHUB_URL = '<YOUR_GITHUB_URL>'
+        BUILD_DIR = '.'
+        CONTAINER_REPO = '<YOUR_REPO_NAME>'
+        CONTAINER_TAG = "${BUILD_NUMBER}"
+        FALCON_CLOUD_REGION = 'us-1'
```

Comment:
Can/should this be a variable? 

If not, most net-new customers are being provisioned in `us-2`.

Reply:
Also, `FALCON_CLOUD_REGION` does not appear to be used anywhere.

Reply:
These are picked up as an environment variable by the python script. I put the variables here at the top specifically so it sticks out so people will know it can be changed.